index.html

```diff
@@ -214,8 +202,10 @@ <h1>About Me</h1>
             <li class="fontaboutmeskills">Gitlab</li>
             <li class="fontaboutmeskills">Gitlab</li>
         </ul>
+    </div>
     </section>
     <footer class="contact">
+        <div class="wrapper2">
         <h2 class="secondary-header">I'm always interested in hearing about new projects, so if you'd like to chat
             please get in touch.</h2>
         <form class="contact-form" action="https://formspree.io/f/xayknejl" method="post">
```

Comment:
- [ ] The contact section `Get in touch` button is currently in the middle of the textarea input instead of being aligned left. The `Get in touch` button should be aligned to the left directly under the input forms. Kindly re-align it to the left as per the Figma design as shown below:
- [ ] Also, the text on the left column should be align left instead of center. please compare the figma design with yours.
👉 Kindly fix these two issues.

Figma Design | Your Design
-----|-----
![image](https://user-images.githubusercontent.com/84156714/186790480-be79de8e-e3e2-45c2-ac24-ed67aa371d40.png) | ![image](https://user-images.githubusercontent.com/84156714/186790789-b469f6fb-63ea-47ec-85e8-98eb642b363a.png)

Reply:
ok i edit it

Reply:
Kindly, in the version I am currently seeing, I still see it centered, please check your code

Reply:
sorry could you please check it, it download the changes